project-plugin/src/main/kotlin/com/android/declarative/project/GenericDeclarativeParser.kt

```diff
@@ -159,6 +227,8 @@ class GenericDeclarativeParser(
                 ?: dslTypeResult.mutableProperties["is${tableKey.capitalized()}"]
             if (mutableProperty != null) {
                 logger.LOG { "F: $tableKey = ${table.get(tableKey)} at $mutableProperty" }
+                // Try setting the property as accessible. "ldlibs" is inaccessible maybe because it is nullable and not initialized?
```

Comment:
that's strange, I think we will need to find out the real underlying reason at some point.